pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -68,6 +69,61 @@
 
 from pulp_ansible.app.tasks.deletion import delete_collection_version, delete_collection
 
+_COLLECTION_ACCESS_POLICY = {
+    "statements": [
+        {
+            "action": ["list", "retrieve"],
+            "principal": "authenticated",
+            "effect": "allow",
+        },
+        {
+            "action": "destroy",
+            "principal": "authenticated",
+            "effect": "allow",
+            "condition": [
+                "has_model_perms:ansible.delete_collection",
+                "has_model_or_obj_perms:ansible.modify_ansible_repo_content",
+            ],
+        },
+        {
+            "action": ["download"],
+            "principal": "authenticated",
```

Comment:
@mdellweg should we allow unauthenticated downloading by default since that's how pulp ansible has worked until now? It doesn't matter to me if we change this now, since this is the default for galaxy, but this is a breaking change for pulp